README.md

```diff
@@ -71,7 +71,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://yacoubou-seidou.github.io/ToDo_List/)
+- [Live Demo Link](https://quiet-cat-6bf6ec.netlify.app/)
```

Comment:
- [ ] Good job adding a `live demo` link , however the content of the link you put is not corresponding to the work you did. I could be due to the `branch` you selected in Netlify when you deployed , Kindly fix that 👍 

![Capture](https://github.com/boniface193/react-final-capstone/assets/80458199/357e746b-9f26-487f-8ce1-bc55ffe87ed8)